.github/workflows/nightly.yml

```diff
@@ -121,7 +121,7 @@ jobs:
     needs: [build, binary, unit-tests]
     strategy:
       matrix:
-        k8s: [1.20.2, 1.19.1, 1.18.8, 1.17.11, 1.16.15]
+        k8s: [1.21.1, 1.20.7, 1.19.11, 1.18.19, 1.17.17, 1.16.15]
```

Comment:
should we remove `1.16.15` ?

Reply:
I was going to keep it until we officially drop it after the release. Having a bigger matrix is not as much of issue now that we have split out the Plus and OSS builds.